ForceFeedback/ForceFeedback.Rules/Extensions/SyntaxNodeExtension.cs

```diff
@@ -0,0 +1,34 @@
+﻿using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+
+namespace ForceFeedback.Rules.Extensions
+{
+    public static class SyntaxNodeExtension
```

Comment:
Code documentation is missing. Please describe the general intent of the class and methods.